modules/server/src/main/resources/db/migration/data/V1__add_protocols_table.sql

```diff
@@ -1,4 +1,6 @@
 CREATE TABLE protocols (
-    id CHARACTER VARYING(255) PRIMARY KEY,
-    protocol BYTEA
+    id CHARACTER VARYING(255) NOT NULL,
+    version INTEGER NOT NULL,
```

Comment:
I don't know if it makes sense, but, shall we consider versions like `1.0.0-b` or `1.0.0-RC1`, or, is it an auto-increment version by now?

Reply:
It's a simple auto-incremental integer while we keep evolving compendium.